.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Docs: https://github.com/marketplace/actions/close-stale-issues
+#
+name: Close Stale Issues
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: 'This issue is being marked stale because it has been open for 80 days with no activity. Please comment if this issue is still affecting you. If there is no change, this issue will be closed in 20 days.'
+          stale-pr-message: 'This PR is being marked stale because it has been open for 80 days with no activity. Please upadte the PR or ask for a fresh review'
+          close-issue-message: 'This issue was closed because it became stale. If the issue is still affecting you, please re-open it, or file a fresh Issue with updated information.'
```

Comment:
From a non-native speaker, this sounds like "as soon as it became stale, it was closed"

I would probably rephrase it as "because it has been stale for 20 days"

Reply:
+1 for more detailed message

Reply:
Ok - but bear in mind that these issues will also have `stale-issue-message` on them by that point